source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5587,6 +5932,33 @@ export class ExecuteAction extends SubmitActionBase {
     }
 }
 
+export class DataQuery extends UniversalAction {
```

Comment:
@jwoo-msft that is because an action isn't necessarily something that happens as a result of a button click. There is no reason why an action shouldn't be executed as a result of different triggered, such as, in this case, the fact that the input's value has changed and new values need to be retrieved from the backend. Besides, an "action" merely defines a contrac, and here the contract is "a Data.Query action contains information on where to retrieve data from (the dataset) and the response must be a set of key/value pairs"

Defining `Data.Query` as an action fits beautifully into the overall model and architecture of the SDK.